onnxruntime/contrib_ops/cuda/bert/packed_attention.h

```diff
@@ -52,6 +52,10 @@ class PackedAttention final : public TrtFusedAttention<T>, public CudaKernel {
   int num_heads_;                          // number of attention heads
   float scale_;                            // scale for softmax. Default is 0.0f, which will be replaced by 1/sqrt(num_heads) later
   std::vector<int64_t> qkv_hidden_sizes_;  // Q, K, V hidden sizes parsed from the qkv_hidden_sizes attribute.
+  bool disable_fused_runner_;
```

Comment:
The parent class (TrtFusedAttention) has same members. Shall we change the member variables of parent class from private to protected?

Reply:
Good suggestion.